installer/dev/project_reunion_definitions.h

```diff
@@ -0,0 +1,64 @@
+﻿  #pragma once
+
+// This header defines the packages included in the installer.
+// By default, the project will use test packages built for verification of this installer.
+
+// For building the installer with non-test packages, include a "project_reunion_definitions_override.h" file.
+// If present, the override header will be loaded to change the package definitions such as the path.
+// Preprocessor definitions used in the override header must be supported by the resource compiler.
+
+// Override header file example:
+//
+// #pragma once
+//
+// #undef PR_FRAMEWORK_X86_PATH
+// #define PR_FRAMEWORK_X86_PATH "custompackages\\my_custom_x86_framework.appx"
+//
+
+// Each package has a package definition listed below.
+// Names for macros are intentionally kept short due to resource compiler length limitations.
+
+// x86 Framework
+#define PR_FRAMEWORK_X86_ID L"PR_FWPACKAGE_X86"
+#define PR_FRAMEWORK_X86_RCID PR_FWPACKAGE_X86
+#define PR_FRAMEWORK_X86_TYPE L"PACKAGE"
+#define PR_FRAMEWORK_X86_RCTYPE PACKAGE
+#define PR_FRAMEWORK_X86_PATH "..\\test\\testpackages\\framework_x86.msix"
```

Comment:
Feels like this data should be some kind of an input parameter.  This mechanism - packing up MSIXes into an SFX that can install them - isn't specific to Project Reunion.

In some future update, read these from a .json file so it's not hard-coded.

Reply:
These are the defaults, for inner-loop testing. There's an override mechanism to use alternate data - as the pipeline does/will to produce the actual signed/release/shipping exe, and others can use to rebuild with their own packages (for those so motivated)

Reply:
Agree, I want to move this over to a build script that generates the .rc and code from a template to include packages defined in a .json file so it can more easily include any package sets, but that ended up being a much larger work item than I had time allotted.  Created #520 to track.